Agile Essay Re-reference.tex

```diff
@@ -0,0 +1,68 @@
+% Please do not change the document class
+\documentclass{scrartcl}
+
+% Please do not change these packages
+\usepackage[hidelinks]{hyperref}
+\usepackage[none]{hyphenat}
+\usepackage{setspace}
+\doublespace
+
+% You may add additional packages here
+\usepackage{amsmath}
+
+% Please include a clear, concise, and descriptive title
+\title{How does agile methodology affect the workload on a team when in practise?}
+
+% Please do not change the subtitle
+\subtitle{COMP150 - Agile Development Practice}
+
+% Please put your student number in the author field
+\author{1608289}
+
+\begin{document}
+
+\maketitle
+
+\abstract{Proposal}
+
+The topic of my proposed essay is how forms of agile methodology affect the workload on a team when in practise in a workplace. In the recent past few years many companies and also development studios have been using scrum as their form of agile methodology. Throughout this essay I will go through both the pros and cons to putting scrum into practise and how it affects the workload on employees. Through doing so I can address many affects it will have on the quality of work, efficiency of work and also morale of a team during the scrum process. 
+
+\section{Introduction}
+The purpose of this essay is to address the effects of the workload on a team when in practise within a game development company. Throughout this essay it will address the pros and cons to how forms of agile methodology can influence the behaviour of a development team. The one agile method this essay will cover is the use of scrum within a working practise. Throughout the past few years the use of agile methodology has become increasingly more popular amongst a varied form of businesses and companies. Specifically, the use of agile methodology within the game development infrastructure. This is due to the simplicity and also benefits a team can take advantage of to increase the quality of work and also the overall efficiency compared to other methods of meeting a deadline with the intended final product. As well as the benefits scrum can provide to a development team, it can also provide a few cons too. 
+
+\section{Trello board}
+"The Development Team usually starts by designing the system and the work needed to convert the Product Backlog into a working product Increment. Work may be of varying size, or estimated effort. However, enough work is planned during Sprint Planning for the Development Team to forecast what it believes it can do in the upcoming Sprint." \cite{scrum alliance}

```

Comment:
Your reference to scrum alliance won't work here. Try: `\cite{scrumalliance}` instead (i.e., identifiers should be one word) and update the `.bib` file correspondingly.